### DISCUSSION TITLE
Welcome to python-opcua Discussions!

### DISCUSSION THREAD

**Discussion by oroulet** [FreeOpcUa/python-opcua#1187] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

**Comment by swamper123** [comment]
First! :^)
For (private) Chat options you may join our gitter channel: https://gitter.im/FreeOpcUa/python-opcua
